cmd/storage/internal/app/app.go

```diff
@@ -25,12 +40,14 @@ type (
 )
 
 type app struct {
-	repo Repo
+	repo    Repo
+	washMap map[string]WashData
```

Comment:
Так нельзя использовать память в многопоточном приложении. Надо ставить блокировки при чтении и изменении данных, посмотри memDB.
map должен быть map[hash]StationData.
Методы Set, Get, SetServiceAmount.
В Set надо проверять, чтобы под разными хешами не было одной и той же станции.
```suggestion
	stations map[string]StationData
```